sdk/arm/resources/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/Azure/azure-sdk-for-go/sdk/arm/resources
+
+go 1.13
+
+require (
```

Comment:
You'll need to update this with the track 1 version that this was carved out from.

Reply:
I have added it